src/matlabcontrol/MatlabProxyFactoryOptions.java

```diff
@@ -32,6 +33,8 @@
 	private final String _licenseFile;
 	private final boolean _useSingleCompThread;
 	private final int _port;
+	private final Writer _inputWriter;
+	private final Writer _errorWriter;
 
```

Comment:
Throughout this PR, I think "InputXXX" should be named "OutputXXX".  A process has "stdout, stderr, and stdin".  In this case, we're reading data from the process, so "stderr" and "stdout" are actually inputs that we're reading, but I still think "stdout" should be named "output" instead of "input".